config.py

```diff
@@ -31,9 +31,11 @@
 XRAY_SUBSCRIPTION_URL_PREFIX = config("XRAY_SUBSCRIPTION_URL_PREFIX", default="").strip("/")
 
 
-TELEGRAM_API_TOKEN = config("TELEGRAM_API_TOKEN", default=None)
-TELEGRAM_ADMIN_ID = config("TELEGRAM_ADMIN_ID", cast=int, default=0)
-TELEGRAM_PROXY_URL = config("TELEGRAM_PROXY_URL", default=None)
+TELEGRAM_API_TOKEN = config("TELEGRAM_API_TOKEN", default="")
+TELEGRAM_ADMINS_ID = config('TELEGRAM_ADMINS_ID', cast=Csv(int), default=[])
```

Comment:
default value must be string, as it causes `AttributeError: 'list' object has no attribute 'read'` when the variable is not set. also I think using regular split would be much simpler than using CSV reader.